Region/RegionHandler.cc

```diff
@@ -881,7 +881,8 @@ bool RegionHandler::GetRegionHistogramData(
         auto t_end_region_histogram = std::chrono::high_resolution_clock::now();
         auto dt_region_histogram =
             std::chrono::duration_cast<std::chrono::microseconds>(t_end_region_histogram - t_start_region_histogram).count();
```

Comment:
Out of curiosity: why do you get the count in microseconds then convert ( * 1e-3) to milliseconds?  You can get the count in milliseconds: `std::chrono::duration_cast<std::chrono::milliseconds>(t_end - t_start).count();`

Reply:
> Out of curiosity: why do you get the count in microseconds then convert ( * 1e-3) to milliseconds? You can get the count in milliseconds: `std::chrono::duration_cast<std::chrono::milliseconds>(t_end - t_start).count();`

if you get it in milliseconds, doesn't it round it to the nearest millisecond?

Reply:
Yes, you are right.